platform/src/main/java/org/hillview/storage/CassandraDatabase.java

```diff
@@ -181,67 +236,60 @@ private void setStoredTableInfo() {
         for (KeyspaceMetadata kMetadata : keyspaces) {
             keyspace = kMetadata.getName();
             tables = this.metadata.getKeyspace(keyspace).getTables();
-            tableNames = tables.stream().map(AbstractTableMetadata::getName)
-                .collect(Collectors.toList());
+            tableNames = tables.stream().map(AbstractTableMetadata::getName).collect(Collectors.toList());
```

Comment:
Move the declaration of the variables here; they do not need to exist outside the loop.